juniper_official/Solutions/Traffic-Blackhole/interface-drops.rule

```diff
@@ -0,0 +1,162 @@
+/*
+ * REQ: COL-1109
+ * Rule to get all the Interface Drops
+ * Upstream application will use these stats to find Blackholes
+ */
+healthbot {
+    topic hardware.fpc {
+        rule interface-drops {
+            keys interface-name;
+            synopsis "Interface Drops";
+            description "Interface Drops";
+            sensor sensor-interface-drops {
+                synopsis "Interface Drops";
+                description "Interface Drops";
+                open-config {
+                    sensor-name /interfaces/interface/;
```

Comment:
This rule needs to be split into 2 rules so that we don't get all the data under /interfaces/interface and reduce the load on the device and ingest micro services. One rule will collected data from "/interfaces/interface/state" hierarchy and the sensor subscription for the rule should be "/interfaces/interface/state".

Second rule will collected the data for field which are at "/interfaces/interface/ethernet/state" hierarchy and the sensor subscription for this rule should be "/interfaces/interface/ethernet/state"

Reply:
Have split this rule into 2: interface-drops.rule that polls /interfaces/interface/state/ and interface-ethernet-drops.rule that polls /interfaces/interface/ethernet/state/